app/controllers/api/v1/items_controller.rb

```diff
@@ -3,6 +3,11 @@ def create
         render json: Item.create(item_params)
     end
 
+    def update
+        update_item = Item.find(params[:id])
```

Comment:
logic uses AR method, need to prioritize this quality in in `controller.rb` files